### PR TITLE
docs: v3 in dropdown

### PR DESCRIPTION
**Pull request by bennypowers**
## What I did

1. adds v3 to the versions dropdown
2. replaces `<select>` with `<pf-dropdown>`

## Notes to Reviewers

1. relies on #2703

### CODE REVIEW

**Comment by zeroedin** [comment]
Should `pf-button` in `pf-dropdown` have a slot instead of the word `Dropdown` hard coded?  /cc @nikkimk 

https://github.com/patternfly/patternfly-elements/blob/d4e5411959d653a03e9aceb5dfaa913d30cfd317/elements/pf-dropdown/pf-dropdown.ts#L87-L96

File: docs/_includes/_nav.njk [hunk 1]
```diff
@@ -12,48 +12,25 @@
...
+          </picture>
         </a>
       </div>
-      {# TODO: implement pf-dropdown
-      <pf-dropdown id="docs-versions-dropdown" aria-label="Docs Versions" label="Docs Versions">
+      <pf-dropdown id="docs-versions-dropdown">
+        <pf-button slot="trigger"
+                   variant="control"
+                   icon="caret-down"
+                   icon-set="fas">Versions</pf-button>
```
Comment (zeroedin) [inline docs/_includes/_nav.njk:22]: Given if the prior comment is correct I think the button is already handled by `pf-dropdown` instead it should have the ability to slot in text into that button

```Suggestion
        <span slot="button-text">Versions</span>
```

Comment (zeroedin) [reply docs/_includes/_nav.njk:22]: Currently this is outputting Dropdown:

image

Comment (zeroedin) [reply docs/_includes/_nav.njk:22]: Helps if I read notes to reviewers huh?  🙈

**Review by zeroedin — APPROVED** [review]
Losing Gravity, Transferring Magnetosphere

**Comment by bennypowers** [comment]
> EDIT: Helps if I read notes to reviewers 🙈 Sorry for the ping ;)
> 
> Should `pf-button` in `pf-dropdown` have a `<slot name="button-text"></slot>` instead of the word `Dropdown` hard coded? /cc @nikkimk
> 
> https://github.com/patternfly/patternfly-elements/blob/d4e5411959d653a03e9aceb5dfaa913d30cfd317/elements/pf-dropdown/pf-dropdown.ts#L87-L96

#2703

**Comment by nikkimk** [comment]
> > EDIT: Helps if I read notes to reviewers 🙈 Sorry for the ping ;)
> > Should `pf-button` in `pf-dropdown` have a `<slot name="button-text"></slot>` instead of the word `Dropdown` hard coded? /cc @nikkimk
> > https://github.com/patternfly/patternfly-elements/blob/d4e5411959d653a03e9aceb5dfaa913d30cfd317/elements/pf-dropdown/pf-dropdown.ts#L87-L96
> 
> #2703

Yes.

**Pull request merged**